etc/sysconfig/pacemaker.in

```diff
@@ -212,6 +212,15 @@
 #
 # Default: PCMK_remote_port="3121"
 
+# PCMK_remote_pid1 (Advanced Use Only)
+#
+# Specify behavior when the Pacemaker Remote process is not running as PID 1.
+# * If set to "full", Pacemaker Remote behaves as though it's running as PID 1.
+# * If set to "vars", Pacemaker Remote loads environment variables from
+#   /etc/pacemaker/pcmk-init.env, while all other PID 1 behavior is disabled.
+#
+# Default: PCMK_remote_pid1="vars"
```

Comment:
This is true, but there's also no special handling of `"vars"` whatsoever. That was by design. `"vars"` behaves exactly like unset -- see PR description of #1891. So it's effectively the default.

My idea here was to treat this as an enumeration and nudge users toward setting either `"vars"` or `"full"` if they set this at all -- pretending that other values are invalid.

Reply:
Unset and anything-but-full are handled differently. If the environment variable is unset, we return immediately without doing anything. If it's anything besides "full", we do just the vars.

Reply:
Since this is a relatively recent addition and an extremely narrow use case, I wouldn't mind making it a true enumeration and coming up with three distinct values, defaulting to one of them.

Reply:
Ah yep, I misread, thanks for catching it.

True enumeration sounds good to me.